rescheduler/rescheduler.go

```diff
@@ -112,6 +123,11 @@ func main() {
 				if len(criticalPods) > 0 {
 					for _, pod := range criticalPods {
 						glog.Infof("Critical pod %s is unschedulable. Trying to find a spot for it.", podId(pod))
+						k8sApp := "unknown"
+						if l, found := pod.ObjectMeta.Labels["k8s-app"]; found {
```

Comment:
I'm not sure whether it's formalized but `k8s-app` is applied to every critical addon. I understand that it's not ideal but I'd like to have some metadata about problems with different addons and haven't found anything better (pod name is changed when a pod is restarted).